include/rdma/fabric.h

```diff
@@ -171,6 +171,7 @@ typedef struct fid *fid_t;
 #define FI_XPU_TRIGGER		(1ULL << 44)
 #define FI_HMEM_HOST_ALLOC	(1ULL << 45)
 #define FI_HMEM_DEVICE_ONLY	(1ULL << 46)
+#define FI_HMEM_DATA_GDR	(1ULL << 47)
```

Comment:
Though this value is equal to FI_HMEM, it shouldn't be a problem. They are not expected to be used in the same way.

Reply:
Probably makes sense to just model it as an alias for `FI_HMEM` in that case

```suggestion
#define FI_HMEM_DATA_GDR	FI_HMEM
```

Reply:
Oh? IMO it is confusing since it is incidental that these 2 flags are equal.

Or maybe I should simply pick a different value to avoid this confusion.

Reply:
Having the same value and being an alias are not the same.  Flags for mr_attr should go into fi_domain.h.  The FI_HMEM_HOST_ALLOC and FI_HMEM_DEVICE_ONLY flags should have gone there (and probably have been given different values).  See comment in fi_domain.h on memory registration flags.  I would not choose a value that conflicts with FI_HMEM.  The possible need to use that flag is too high.

In general, flags for functions typically start at bit 59 and go down from there, skipping any conflicts (mostly with caps).  So, I'd make this bit 59, since that's still available.